gorush/TAG

```diff
@@ -1 +1 @@
-1.12.0.2
+1.13.0.3
```

Comment:
You may reset the image revision to 1 when the other versions change.
```suggestion
1.13.0.1
```

Reply:
- v1.13.0.1 is used by https://github.com/cybozu/neco-containers/pull/440
- v1.13.0.2 is used by https://github.com/cybozu/neco-containers/pull/464

Reply:
Oh, so why was this line `1.12.0.2`?

Reply:
After update gorush v1.13.0, I happend to gorush degrade. So I downgrade version gorush v1.12.0

Ref: https://github.com/cybozu/neco-containers/pull/468

After all, I found workaround of gorush degrade. In maneki-apps I use gorush v1.13.0